tools/settings.py

```diff
@@ -71,6 +51,42 @@
 # mbed.org username
 MBED_ORG_USER = ""
 
+
+##############################################################################
+# User Settings (file)
+##############################################################################
+try:
+    # Allow to overwrite the default settings without the need to edit the
+    # settings file stored in the repository
+    from mbed_settings import *
+except ImportError:
+    pass
+
+
+##############################################################################
+# User Settings (env vars)
+##############################################################################
+_ENV_PATHS = ['ARM_PATH', 'GCC_ARM_PATH', 'GCC_CR_PATH', 'IAR_PATH']
```

Comment:
True, but if someone were to make a tool that used them, we would get unexpected behavior. I don't think anyone will klober our MBED_ prefixed environment variables.